lang/en.json

```diff
@@ -1,6 +1,6 @@
 {
     "first-slide-title": "Small Size League World Champion",
-    "first-slide-desc": "1st place on RoboCup 2022 - Division B",
+    "first-slide-desc": "1st place on RoboCup 2023 - Division B",
 
     "second-slide-title": "Small Size League Latin American Champion!",
     "second-slide-desc": "1st place on Latin American Robotics Championship 2021 and 3rd place on RoboCup 2021",
```

Comment:
Update with the latest results. Suggestion: "1st place on Latin American Robotics Competition 2022 and 1st place on RoboCup 2023"